cli/admin/src/main/java/io/pravega/cli/admin/AdminCommand.java

```diff
@@ -438,5 +444,26 @@ private ConnectionPool createConnectionPool() {
         return new ConnectionPoolImpl(clientConfig, new SocketConnectionFactoryImpl(clientConfig));
     }
 
+    /**
+     * Creates the file (and parent directory if required).
+     *
+     * @param fileName The name of the file to create.
+     * @return A {@link File} object representing the filename provided.
+     * @throws FileAlreadyExistsException if the file already exists, to avoid any accidental overwrites.
+     * @throws IOException if the file/directory creation fails.
+     */
+    protected File createFileAndDirectory(String fileName) throws FileAlreadyExistsException, IOException {
```

Comment:
No need to throw FileAlreadyExistsException since it derives from IOException.
This could be a static method in utils.

Reply:
done